src/main/java/util/RandomBinaryNumbers.java

```diff
@@ -0,0 +1,25 @@
+package util;
+
+import static java.util.stream.Collectors.toList;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class RandomBinaryNumbers {
+    private static final int START_INCLUSIVE = 0;
+    private static final int END_INCLUSIVE = 1;
+
+    private RandomBinaryNumbers() {
+    }
+
+    public static List<Integer> pickNumbers(int size) {
+        return IntStream.range(0, size)
+                .mapToObj(i -> getBinaryNumber())
```

Comment:
```suggestion
                .mapToObj(i -> Randoms.pickNumberInRange(0, 1))
```
1. 이전에 남긴 코멘트 중에서 `Ladder.create()` 를 메서드 분리할 필요가 있을까? 를 고민해본 적 있었죠? getBinaryNumber() 역시 보면서 같은 생각이 들었어요.
2. 이미 클래스명이 `RandomBinaryNumbers` 라서 꼭  0 과 1 을 `START_INCLUSIVE` 와 `END_INCLUSIVE` 라고 나타내지 않아도 될 것 같아요. 이 상황에서는 0 과 1 그 자체가 스스로를 너무 잘 설명하고 있거든요. `INCLUSIVE` 라는 표현을 넣고 싶었다면 그럴 순 있다고는 생각해요 🤔